Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -484,6 +484,127 @@ class ProductStoreTests: XCTestCase {
 
         wait(for: [backgroundSaveExpectation], timeout: Constants.expectationTimeout)
     }
+
+    // MARK: - ProductAction.searchProducts
+
+    /// Verifies that `ProductAction.searchProducts` effectively persists the retrieved products.
+    ///
+    func testSearchProductsEffectivelyPersistsRetrievedSearchProducts() {
+        let expectation = self.expectation(description: "Search Products")
+        let store = ProductStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
+        let expectedProduct = sampleProduct()
+
+        network.simulateResponse(requestUrlSuffix: "products", filename: "products-load-all")
```

Comment:
Similar to the PR in networking, I think it might be worth specialising the mocked response. What do you think?